hippod/hippod_login.py

```diff
@@ -0,0 +1,173 @@
+
+import os
+
+import datetime
+
+import json
+
+from aiohttp import web
+
+
+# Customize the cookie
+# this is more convient way to modify cookie.
+COOKIE_NAME = "OldTamil"
+COOKIE_VALUE = 0
+
+
+# Define host and port.
+HOST = '127.0.0.1'
+PORT = 8080
+
+# Add required files.
+LOGIN_HTML = 'login_system/login.html'
+SITE_HTML = 'login_system/site.html'
+REDIRECT_HTML = 'login_system/redirect.html'
+CONFIG_FILE = 'login_system/configuration.json'
+
+
+# We use ERROR for server_error web response.
+ERROR = '''<!DOCTYPE html>
```

Comment:
No, this is a static error page, should no assets where all HTML files are stored